_posts/en/newsletters/2020-08-19-newsletter.md

```diff
@@ -0,0 +1,164 @@
+    thread.
+
+- **Proposed BIP for P2P protocol feature negotiation:** Suhas Daftuar
+  [posted][daftuar negotiation] a suggestion to create a [separate
+  BIP][bip-negotiation] for the P2P feature negotiation method that
+  became a part of [BIP339][] (see [Newsletter #87][news87
+  negotiation]).  The protocol change is simple: when a node receives a
+  P2P `version` message from a newly-connected peer, the node should
+  ignore any messages it doesn't understand until a `verack` message has
+  been received from that peer.  Those messages, like the new BIP339
+  `wtxidrelay` message, may signal what features the remote peer
+  supports.
+
+    This behavior is already implemented in an unreleased development
+    version of Bitcoin Core.  If the maintainers of other
+    implementations, or anyone else, opposes this change, please respond
+    to the mailing list thread.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.11.0-beta.rc4][lnd 0.11.0-beta] is a release candidate for the
+  next major version of this project.  It allows accepting [large
+  channels][topic large channels] (by default, this is off) and contains
+  numerous improvements to its backend features that may be of interest
+  to advanced users (see the [release notes][lnd 0.11.0-beta]).
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19658][] Allow RPC to fetch all addrman records and add records to addrman FIXME:dongcarl
+
+- [Bitcoin Core #18654][] adds a new `psbtbumpfee` RPC that takes the
+  txid of a transaction currently in the local node's mempool and
+  creates a [PSBT][topic psbt] that increases its feerate (either by an
+  automatic amount or by an amount specified via a parameter).  Other
+  RPCs or external tools can then sign and finalize the PSBT.
+
+    The existing `bumpfee` RPC previously created PSBT fee bumps for
+    watch-only wallets (see [Newsletter #80][news80 bumpfee]).  This
+    behavior is now deprecated and a message will be printed to tell
+    those users to use `psbtbumpfee`.  Use of `bumpfee` with regular
+    key-containing wallets is unaffected.
+
+- [Bitcoin Core #19070][] causes Bitcoin Core to advertise its support
+  for [compact block filters][topic compact block filters] if the
+  `peerblockfilters` configuration option has been enabled (default is
```

Comment:
both `-peerblockfilters` and `-blockfilterindex` need to be enabled, otherwise bitcoind raises an init error

maybe: "if the `peerblockfilters` and `blockfilterindex` options have been set (default is off)."